packages/transactions/tests/clarity.test.ts

```diff
@@ -678,4 +685,16 @@ describe('Clarity Types', () => {
       );
     });
   });
+
+  describe('Clarity type narrowing', () => {
+    it('can narrow strings', () => {
+      const vUint = Cl.uint(1) as ClarityValue;
+      expect(isClarityType(vUint, ClarityType.UInt)).toBeTruthy();
+      expect(isClarityType(vUint, ClarityType.Int)).toBeFalsy();
+
+      const vInt = Cl.int(1) as ClarityValue;
+      expect(isClarityType(vInt, ClarityType.Int)).toBeTruthy();
+      expect(isClarityType(vInt, ClarityType.UInt)).toBeFalsy();
```

Comment:
This tests the runtime check, but doesn't test the type assertion. I.e. something like this:
```ts
const intTest: IntCV = vInt;
```

Reply:
Good idea 👍🏻

Reply:
Good point! Done